### DISCUSSION TITLE
Export media to disc

### DISCUSSION THREAD

**Discussion by wizardion** [photoprism/photoprism#2340] [General]
Hello, I'm interested in this feature if it is planned or not but, I didn't find this in the documentation. 
I would like to have all processed/converted files, including photos and videos (transcoded to avc extension), be exported to a disc as a post-processed media. Is there anything you can guys suggest to me to achieve this goal?

**Comment by graciousgrey** [comment]
Hello :)

Converted files are stored in storage/sidecar, you can configure the location of the sidecar folder in your docker-compose.yml 

https://docs.photoprism.app/user-guide/advanced/folders/#sidecar

**Comment by Egor-S-N** [comment]
hello
